### PR TITLE
Debug for Account

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Derive prints the full userdata vec which is questionably useful.

#### Summary of Changes

Custom debug formatter.

Fixes #

### CODE REVIEW

File: sdk/src/account.rs [hunk 1]
```diff
@@ -14,6 +15,19 @@ pub struct Account {
     pub executable: bool,
 }
 
+impl fmt::Debug for Account {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(
+            f,
+            "Account {{ tokens: {} userdata.len: {} owner: {} executable: {} }}",
+            self.tokens,
+            self.userdata.len(),
```
Comment (garious) [inline sdk/src/account.rs]: How about a base58 string instead for userdata? Check out the instance for Pubkey.

Comment (sakridge) [reply sdk/src/account.rs]: how is it helpful though?

Comment (garious) [reply sdk/src/account.rs]: You can diff it.  Also, the extremely motivated can copy that string and pass it to a utility that parses it to recreate the original account.

Comment (sakridge) [reply sdk/src/account.rs]: I feel like if you want something useful, you add another debug at some point where you have deserialized the userdata and then print something that makes sense depending on the account type. It's way less work to do that than it is to write a whole new utility to parse strings that you copy and paste and better debug experience.

Comment (garious) [reply sdk/src/account.rs]: The default Debug instances include *all* the information, but it's too verbose.  The next closest thing is *all* the information in a more compact way, like base58. If that's still too verbose (which it's generally not), then printing just the length makes sense to me.

Comment (garious) [reply sdk/src/account.rs]: A hex string is another, probably better option.

Comment (sakridge) [reply sdk/src/account.rs]: Here's a compromise, up to the first 64 bytes hex-formatted if it's not 0 and always include the length.

**Pull request merged**